tdrs-frontend/src/components/FeedbackForm/FeedbackForm.jsx

```diff
@@ -0,0 +1,171 @@
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react'
+import axios from 'axios'
+import FeedbackRadioSelectGroup from '../FeedbackRadioSelectGroup'
+import { feedbackPost } from '__mocks__/mockFeedbackAxiosApi'
+
+const FeedbackForm = ({ onFeedbackSubmit }) => {
+  const [isAnonymous, setIsAnonymous] = useState(false)
+  const [selectedRatingsOption, setSelectedRatingsOption] = useState(undefined)
+  const [feedbackMessage, setFeedbackMessage] = useState('')
+  const [error, setError] = useState(false)
+
+  const resetStatesOnceSubmitted = () => {
+    setSelectedRatingsOption(undefined)
+    setFeedbackMessage('')
+    setError(false)
+    setIsAnonymous(false)
+  }
+
+  function sanitizeInput(input) {
+    const div = document.createElement('div')
+    div.textContent = input
+    return div.innerHTML
+  }
+
+  const submitForm = (data) => {
+    return axios.post('/api/userFeedback/', data)
```

Comment:
maybe a note for the api implementer, we probably want to use `axiosInstance` here to carry over the authentication credentials. related: #3722